src/pyramid/config/adapters.py

```diff
@@ -80,7 +80,7 @@ def _derive_predicate(self, predicate):
 
         if eventonly(predicate):
 
-            def derived_predicate(*arg):
+            def derived_predicate(*arg):  # noqa: F811
```

Comment:
Probably better to fix the lint error rather than ignoring it.  Maybe:

```python
if not eventonly(predicate):
    return predicate

def derived_predicate(*arg):
     return predicate(arg[0])

return derived_predicate
```

Reply:
Definitely, went for `else` clause.